edu.berkeley.BOINC.yml

```
@@ -1,6 +1,6 @@
 app-id: edu.berkeley.BOINC
 runtime: org.gnome.Platform
-runtime-version: "46"
+runtime-version: "48"
```

```suggestion
runtime-version: "48"
##### test comment
```
This is only a test comment to re-trigger the build process, because the old build is no longer available (404) for testing. Will be removed immediately